design/design-doc-observe-only-resources.md

```diff
@@ -0,0 +1,293 @@
+	Update(ctx context.Context, mg resource.Managed) (ExternalUpdate, error)
+	Delete(ctx context.Context, mg resource.Managed) error
+}
+```
+
+We will leverage the fact that we have an already implemented Observe method for all managed resources by calling only
+it when the Management Policy is set to ObserveOnly. This will require minor modifications in the Managed Reconciler
+code that will return early in the reconcile loop and prevent invocation of the other methods, namely, Create, Update
+and Delete. These modifications will implement the following logic at a high level:
+
+Right after the `Observe` method invocation, if `ObserveOnly` :
+
+- Return error if the resource does not exist
+- Publish connection details
+- Call `client.Update` if the resource was late initialized
+- Report success and return early
+
+#### Feature Gating
+
+Similar to all other new features being added to Crossplane, we will ship this new policy as an alpha feature that will
+be off by default and will be controlled by `--enable-alpha-management-policies` flag.
+
+This will not prevent the field from appearing in the schema of the managed resources. However, we will ignore the
+`spec.managementPolicy` when the feature is not enabled.
+
+#### Deprecation of `deletionPolicy`
+
+With the new `managementPolicy` covering the existing `deletionPolicy`, we will deprecate the latter in favor of the
+former.
+
+Until we drop the `deletionPolicy` from the schema altogether, we need to be careful with the conflicting combinations
+shown in the below table which only exists with deletion:
+
+| Deletion Policy | Management Policy | Should Observe? | Should Create? | Should Update? | Should Delete? |
+| --- | --- | --- | --- | --- | --- |
+| Delete | Full | Yes | Yes | Yes | Yes |
+| Orphan | ObserveCreateUpdate | Yes | Yes | Yes | No |
+| Delete | ObserveOnly | Yes | No | No | Conflict (No) |
+| Orphan | Full | Yes | Yes | Yes | Conflict (No) |
+| Delete | ObserveCreateUpdate | Yes | Yes | Yes | Conflict (No) |
+| Orphan | ObserveOnly | Yes | No | No | No |
+
+For conflicting cases, we will decide based on the non-default configuration and not delete the external resource.
+This way, we will also err on the side of caution by leaving the actual resource. However, this comes at the cost of
+some possible confusion for users focusing on only one of the policies.
+
+Another solution could be simply throwing an error and preventing reconciliation during conflict. This would be more
+explicit but would require some manual actions and degraded UX for the usage of the feature, for example:
+
+- Creating an ObserveOnly resource will require both setting `managementPolicy` to `ObserveOnly` and `deletionPolicy`
+to `Orphan` .
+- If there are existing resources with `deletionPolicy: Orphan` when the feature is enabled, they will start failing to
+reconcile until their `managementPolicy`’s updated to `ObserveCreateUpdate`.
```

Comment:
This section starts off by stating what the behavior will be when there is a conflicting case but then goes into an alternative solution, leaving the reader unclear as to what the actual recommendation of the design doc is. 

Propose that it is reframed so that it is clear which approach the design doc puts forward and lists the other as something that was given consideration but not chosen.

Reply:
I'll reframe accordingly and make it clear that we will go with _not deleting without throwing an error_  when there is a conflict.